packages/core/tsconfig.types.json

```diff
@@ -4,7 +4,9 @@
     "emitDeclarationOnly": true,
     "outDir": "types",
     "esModuleInterop": true,
-    "resolveJsonModule": true
+    "resolveJsonModule": true,
+    "module": "CommonJS",
```

Comment:
any reason why we are adding commonJS? I think we have ES2022 in the other packages

Reply:
should we use ES2022 instead to have uniformity accross the packages?